REST/Request.cc

```diff
@@ -387,6 +387,11 @@ namespace litecore { namespace REST {
     }
 
 
+    void RequestResponse::onClose(std::function<void()> callback) {
+        _socket->onClose(move(callback));
```

Comment:
Does this move have any effect if the function `onClose` doesn't use the `&&` syntax?  I think if it did a copy would be saved.